### ISSUE TITLE
Failure on Windows

### ISSUE THREAD

**Issue by imuzzolini** [ergoithz/browsepy#12]
I am using Windows 10 and python 2.7. I created a virtual environment and then installed browsepy with the following:
```
virtualenv venv
venv\Scripts\activate.bat
pip install browsepy
```
And then launched with:
```
python -m browsepy 0.0.0.0 8080 C:\Users\user\Desktop
```
*note:* the directory `C:\Users\user\Desktop` contains several files.

I then opened a browser on `http://localhost:8080/` and python send back the rendered HTML page with the correct name of the directory but I cannot see any file. The log information is as follows:
```
 * Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
127.0.0.1 - - [24/Feb/2017 12:46:49] "GET / HTTP/1.1" 200 -
Error on request:
Traceback (most recent call last):
  File "C:\Users\user\experiments\venv\lib\site-packages\werkzeug\serving.py", line 205, in run_wsgi
    execute(self.server.app)
  File "C:\Users\user\experiments\venv\lib\site-packages\werkzeug\serving.py", line 195, in execute
    for data in application_iter:
  File "C:\Users\user\experiments\venv\lib\site-packages\werkzeug\wsgi.py", line 703, in __next__
    return self._next()
  File "C:\Users\user\experiments\venv\lib\site-packages\werkzeug\wrappers.py", line 81, in _iter_encoded
    for item in iterable:
  File "C:\Users\user\experiments\venv\lib\site-packages\flask\helpers.py", line 130, in generator
    for item in gen:
  File "C:\Users\user\experiments\venv\lib\site-packages\jinja2\environment.py", line 1045, in generate
    yield self.environment.handle_exception(exc_info, True)
  File "C:\Users\user\experiments\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\user\experiments\venv\lib\site-packages\browsepy\templates\browse.html", line 58, in top-level template code
    >{{ text }}</a>
  File "C:\Users\user\experiments\venv\lib\site-packages\browsepy\templates\base.html", line 13, in top-level template code
    {% block content %}{% endblock %}
  File "C:\Users\user\experiments\venv\lib\site-packages\browsepy\templates\browse.html", line 99, in block "content"
    {% block content_table %}
  File "C:\Users\user\experiments\venv\lib\site-packages\browsepy\templates\browse.html", line 113, in block "content_table"
    {% for f in file.listdir(sortkey=sort_fnc, reverse=sort_reverse) %}
  File "C:\Users\user\experiments\venv\lib\site-packages\browsepy\file.py", line 628, in listdir
    data = sorted(self._listdir(), key=sortkey, reverse=reverse)
  File "C:\Users\user\experiments\venv\lib\site-packages\browsepy\file.py", line 613, in _listdir
    kwargs['stats'] = entry.stats()
AttributeError: 'scandir.DirEntry' object has no attribute 'stats'
```
It feels like `stats` is not available on Windows.

**Comment by ergoithz** [comment]
Thanks for reporting, it seems to be a typo (using non-existing method stats instead of stat), I'll fix it soon.

**Comment by ergoithz** [comment]
I've just added some NT-specific tests, fixed this issue, and [released 0.5.2](https://github.com/ergoithz/browsepy/releases/tag/0.5.2). Thanks for your report!
